gen-bundle-image.sh

```diff
@@ -19,6 +19,7 @@ function set_bundle_variables {
 
    vfkit_bundle=crc${bundlePreset}_vfkit_${version}_amd64.crcbundle
    libvirt_bundle=crc${bundlePreset}_libvirt_${version}_amd64.crcbundle
+   libvirt_bundle_arm64=crc${bundlePreset}_libvirt_${version}_arm64.crcbundle
```

Comment:
ahh, so we don't want to generate arm64 libvirt bundles only for `okd` by looking at the changes, down.. so maybe this should be inside the `okd` check above at L17

Reply:
@anjannath done.